src/pip/_internal/build_env.py

```diff
@@ -48,7 +48,10 @@ def _create_standalone_pip() -> Iterator[str]:
     The zip file's content is identical to the currently-running pip.
     It will be used to install requirements into the build environment.
     """
-    source = pathlib.Path(pip_location).resolve().parent
+    # Replit mod: the pip binary is a symlink to the content-addressable cache,
+    # so the original `.resolve()` was causing the zip to have some _very_
+    # unexpected contents.
+    source = pathlib.Path(pip_location).parent
```

Comment:
Maybe we should still do something about resolving the location to absolute path, but ignore the symlink.

Reply:
that's already absolute! https://docs.python.org/3.4/whatsnew/3.4.html#other-language-changes

> Module `__file__` attributes (and related values) should now always contain absolute paths by default, with the sole exception of `__main__.__file__` when a script has been executed directly using a relative path.

Since this is not `__main__.__file`, this path is already absolute, thanks to L19 being `from pip import __file__ as pip_location`.